samples/sampler/stories/qa/Scroller.js

```diff
@@ -243,7 +252,9 @@ export const EditableList = (args) => {
 			direction="horizontal"
 			editable={{
 				centered: args['editableCentered'],
-				onComplete: handleComplete
+				editableWrapperCss: css,
```

Comment:
let's rename this to `css` and polyfill public class names.

Reply:
done